modules/sql-engine/src/integrationTest/sql/types/time/time_parsing.test

```diff
@@ -25,5 +25,5 @@ SELECT '14:42:04.999'::TIME(3)::VARCHAR
 query I
 SELECT '14:42:04.999'::TIME(6)::VARCHAR
 ----
-14:42:04.999000
+14:42:04.999
```

Comment:
Why does precision is ignored?

Reply:
Precision isn't ignored, we just don't add trailing zeros now.
I copied the original test to make it more clear.